tests/Unit/PointwiseFunctions/AnalyticSolutions/Test_Errors.cpp

```diff
@@ -0,0 +1,63 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Evolution/ComputeTags.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/System.hpp"
+#include "Helpers/DataStructures/DataBox/TestHelpers.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/Errors.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/GaugeWave.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/WrappedGr.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/Tags.hpp"
+#include "Time/Tags.hpp"
+#include "Utilities/TMPL.hpp"
+
+SPECTRE_TEST_CASE("Unit.AnalyticSolutions.Errors",
+                  "[Unit][PointwiseFunctions]") {
+  static constexpr size_t volume_dim = 1;
+  using system = GeneralizedHarmonic::System<volume_dim>;
```

Comment:
I suggest to define a few tags for the purpose of this test and make it independent of the GH and evolution code. The code that's being tested only needs a few tags and their `::Tags::Analytic` counterparts in the box, so there's no need to involve the GH system and analytic solutions. You don't need to use the `evolution::Tags::AnalyticCompute` either, just put simple tags in the box.

Reply:
I thought about this, but I decided that I specifically want to test that this machinery works with with AnalyticCompute and GH. The test you suggest would not have caught the problem with WrappedGr<> that this PR fixes (it didn't have a tags alias), e.g. Also, I want to make sure that if AnalyticCompute ever changes that it won't break DifferenceFromAnalytic, whereas if I add the Analytic<> tags by hand, this might happen

Reply:
I disagree. This PR implements a unit of functionality that should be tested. If you want to test something with GH and WrappedGr then I suggest you add it to the WrappedGr tests.